### DISCUSSION TITLE
CORS ERROR WHILE ACCESSING REMIX BUILD FILES HOSTED ON HEROKU

### DISCUSSION THREAD

**Discussion by mvtdev1050** [remix-run/remix#7709] [Q&A]
What I'm doing is I'm creating a SHOPIFY app using remix template.

I've hosted that app on the heroku.

But when I use one of its routes as app proxy in shopify 

It giving me a cors error whilr fetching the build files.
In ss there is a cloudflare on whoch shopify host for development and at there I;m getting the same cors issue while fetching build files

![image](https://github.com/remix-run/remix/assets/28701205/ab4ffaea-db82-4a6d-9c2c-16a7ef0cdb39)

What I want is that where can I add cors headers so the app hosted on any server will be accessible by origin I've passed in the headers.

I have tried passing in entery server file but all they add headers for text/html files request 

Where to add cors headers for the application/javascript files.

I've also used handleDataRequest but it's not working

**Comment by sydneyroosters** [comment]
@mvtdev1050  Did you ever solve this issue?
